### ISSUE TITLE
Problem choosing a month / year in custom range

### ISSUE THREAD

**Issue by wesleifreitas** [mominsamir/smDateTimeRangePicker#134]
**Steps to the problem:**

1. Access http://mominsamir.github.io/smDateTimeRangePicker/#!/range-picker-demo and click on input
![untitled-1 fw](https://cloud.githubusercontent.com/assets/6631277/23802484/4ae24eb8-0592-11e7-9ab8-8fa36cd12b36.png)

2. Click in **CUSTOM RANGE**
![untitled-2 fw](https://cloud.githubusercontent.com/assets/6631277/23802607/9b401854-0592-11e7-866e-5451bb150a32.png)

3. Click in month-year
![untitled-3 fw](https://cloud.githubusercontent.com/assets/6631277/23805965/af0ebff0-059e-11e7-9510-ff84d3720479.png)

3. Click in month of other year (e.g. may/**2018**)
![untitled-4 fw](https://cloud.githubusercontent.com/assets/6631277/23806021/de3876b8-059e-11e7-9dde-4bf0fc054fa2.png)

4. Problem! may(**2017**)
![untitled-5 fw](https://cloud.githubusercontent.com/assets/6631277/23806046/f3ce2950-059e-11e7-8ba2-7610ef692070.png)

**Comment by mominsamir** [comment]
fixed this issue, pushing fix

**Comment by wesleifreitas** [comment]
Hi @mominsamir 
Has the issue been fixed? Otherwise I can try to fix

**Comment by mominsamir** [comment]
Yes this is fixed but not pushed yet, i will try to push it tonight

**Comment by wesleifreitas** [comment]
Ok 👍 

I found another problem with the UTC moment, I sent a pull request ;)
